engine/components-api/src/main/java/com/cloud/storage/StorageManager.java

```diff
@@ -101,6 +101,9 @@
             ConfigKey.Scope.Cluster,
             null);
 
+    ConfigKey<Integer> PRIMARY_STORAGE_DOWNLOAD_WAIT = new ConfigKey<Integer>("Storage", Integer.class, "primary.storage.download.wait", "10800",
```

Comment:
isn't such a variable already in place somewhere? there are other places where it would have use. If not, I think it should be applied to the initial deploy equally.

Reply:
`Config.PrimaryStorageDownloadWait` has this configuration indeed.

I can pick from there then and removed this one. However, on the other hand, `Config.PrimaryStorageDownloadWait` is deprecated. Should I use `Config.PrimaryStorageDownloadWait`, or remove the `Config.PrimaryStorageDownloadWait` and keep with the `new ConfigKey<Integer>...`?

Reply:
I'd say keep your new one and find usages of the old one to unify them. If need be move the key to a more centralised place.